src/Mutations/addReports.js

```diff
@@ -5,7 +5,7 @@ export const ADD_REPORT = gql`
     updateSqueak(input: { id: $id, report: $report}) {
```

Comment:
I wonder if you need a different mutation constant for approve vs add reports if it's the same format but just different variable values? Could they be solidated to an UPDATE_SQUEAK mutation where you pass in different values depending on what you're updating?